examples/v2/project_creation/project.py.schema

```diff
@@ -78,6 +78,13 @@ properties:
                 item:
                   type: string
                   pattern: "^(user|group|domain|serviceAccount):"
+  set-dm-service-account-as-owner:
+    description: >
+      Set true so that the service account that Deployment Manager will use
```

Comment:
Consider re-wording this to: "If set to true, the service account that Deployment Manager will use in the newly created project  will be made Owner. This would allow Deployment Manager to set IAM policies within this project."

Reply:
Done